### PR TITLE
Drop openshift-controller-manager staging repo in favor of published repo

### PR DESCRIPTION
**Pull request by sttts**
Published repo is at https://github.com/openshift/openshift-controller-manager.

### CODE REVIEW

**Comment by deads2k** [comment]
ec2

/retest

**Comment by sttts** [comment]
Random e2e tests.

/retest

File: test/extended/builds/controllers.go [hunk 1]
```diff
@@ -19,7 +18,25 @@ import (
 	imagev1 "github.com/openshift/api/image/v1"
 	buildv1clienttyped "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
 	imagev1clienttyped "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
-	"github.com/openshift/openshift-controller-manager/pkg/build/buildutil"
+)
+
+const (
```
Comment (tnozicka) [inline test/extended/builds/controllers.go]: looks api-ish

File: test/extended/builds/controllers.go [hunk 2]
```diff
@@ -127,7 +144,7 @@ func RunImageChangeTriggerTest(t testingT, clusterAdminBuildClient buildv1client
 		registryHostname = "registry:8080"
 	)
 
-	testutil.SetAdditionalAllowedRegistries(registryHostname)
+	//testutil.SetAdditionalAllowedRegistries(registryHostname)
```
Comment (tnozicka) [inline test/extended/builds/controllers.go]: do we have a tracker for a follow up?

File: test/extended/image_ecosystem/mysql_replica.go [hunk 3]
```diff
@@ -1,245 +0,0 @@
-package image_ecosystem
```
Comment (tnozicka) [inline test/extended/image_ecosystem/mysql_replica.go]: a follow up ticket for these 2?

File: test/extended/templates/helpers.go [hunk 4]
```diff
@@ -137,9 +136,9 @@ func dumpObjectReadiness(oc *exutil.CLI, templateInstance *templatev1.TemplateIn
 	fmt.Fprintf(g.GinkgoWriter, "dumping object readiness for %s/%s\n", templateInstance.Namespace, templateInstance.Name)
 
 	for _, object := range templateInstance.Status.Objects {
-		if !controller.CanCheckReadiness(object.Ref) {
-			continue
-		}
+		//if !controller.CanCheckReadiness(object.Ref) {
```
Comment (tnozicka) [inline test/extended/templates/helpers.go]: folloup up ticket?

**Comment by tnozicka** [comment]
link a follow up ticket to bring back the test / commented code and I can tag

File: test/extended/image_ecosystem/mysql_replica.go (hunk 3 above)
Comment (sttts) [reply test/extended/image_ecosystem/mysql_replica.go]: Done: https://bugzilla.redhat.com/show_bug.cgi?id=1731892

File: test/extended/builds/controllers.go (hunk 2 above)
Comment (sttts) [reply test/extended/builds/controllers.go]: also added to https://bugzilla.redhat.com/show_bug.cgi?id=1731892

**Comment by deads2k** [comment]
I think it's fixed here: https://github.com/openshift/origin/pull/23447

**Comment by adambkaplan** [comment]
fyi working on fixing `e2e-aws-builds` - a bunch of tests reference an imagestream tag that is being removed in 4.2

**Comment by sttts** [comment]
```
[sig-auth] ServiceAccounts should allow opting out of API token automount [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s] expand_more	10m5s
[Area:Networking] multicast when using one of the plugins 'redhat/openshift-ovs-multitenant, redhat/openshift-ovs-networkpolicy' should allow multicast traffic in namespaces where it is enabled [Suite:openshift/conformance/parallel] expand_more	56s
node client cert requests armoring: deny pod's access to /config/master API endpoint [Suite:openshift/conformance/parallel] expand_more
```

/retest

/cc @squeed I would bet I saw that multicast test before as flake in this PR.

**Comment by squeed** [comment]
> /cc @squeed I would bet I saw that multicast test before as flake in this PR.

Yup, that's a weird flake caused by something that broke PLEG.

**Pull request closed without merging**